transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannel.java

```diff
@@ -427,48 +411,31 @@ public EpollDatagramChannelConfig config() {
 
     @Override
     protected void doDisconnect() throws Exception {
+        InetSocketAddress address = remoteAddress();
+        socket.disconnect();
         connected = false;
+        if (address != null) {
```

Comment:
@normanmaurer - https://github.com/netty/netty/pull/6193#discussion_r115565539 has been addressed ... but I'm still wondering if we should unconditionally set active to false here instead of depending on `remoteAddress()`?